addons/im_livechat/static/src/models/messaging_notification_handler.js

```diff
@@ -5,19 +5,6 @@ import { patchRecordMethods } from '@mail/model/model_core';
 import '@mail/models/messaging_notification_handler';
 
 patchRecordMethods('MessagingNotificationHandler', {
-    /**
-     * @override
-     * @param {object} settings
-     * @param {boolean} [settings.is_discuss_sidebar_category_livechat_open]
-    */
-    _handleNotificationResUsersSettings(settings) {
-        if ('is_discuss_sidebar_category_livechat_open' in settings) {
-            this.messaging.userSetting.update({
-                is_discuss_sidebar_category_livechat_open: settings.is_discuss_sidebar_category_livechat_open,
-            });
-        }
-        this._super(settings);
```

Comment:
confusing commit split, this should be removed in the commit introducing the computes